yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1101,13 +1101,17 @@ func isEndOfSqlStmt(line string) bool {
 	/*	checking for string with ending with `;`
 		Also, cover cases like comment at the end after `;`
 		example: "CREATE TABLE t1 (c1 int); -- table t1" */
+	line = strings.TrimRight(line, " ")
+	if line[len(line)-1] == ';' {
+		return true
+	}
 
-	cmtStartIdx := strings.Index(line, "--")
+	cmtStartIdx := strings.LastIndex(line, "--")
 	if cmtStartIdx != -1 {
 		line = line[0:cmtStartIdx] // ignore comment
 		line = strings.TrimRight(line, " ")
 	}
-	return strings.Contains(line, ";")
```

Comment:
Thinking of some more cases:

1. DDLs ending with multi-line SQL comments
```
create table temp (
id int,
name varchar
); /* multi-line sql comment*/
```

2. DDLs with multi-line SQL comments in between
```
create table temp (
/* multi-line sql comment*/ id int,
name varchar
);
```

I don't think these are handled before this change also.

Reply:
Good point about multi-line comments. 
```
create table temp (
id int,
name varchar
); /* multi-line sql comment*/
```
Surprisingly, this works right now. Not because it succesfully determines it as the end of an sql statement; it does not. It just moves on, until it finds the end of the next statement. (essentially two statements get merged into one, and that's okay because it's valid sql).

Reply:
@sanyamsinghal the fix for multi-line comments is not straight-forward. We can easily detect multi-line comments as well. However, consider the following case 
```
1: CREATE TABLE
2: ...
3 ); /* multi line 
4: comment */
```

Here we can detect that line 3 is the end of the CREATE TABLE stmt, however pg-parser and analyze-schema then report it the statement (lines 1-3) as a PG unsupported syntax, because the multi-line comment is not closed properly. 

I would suggest that we pick this task up later. lmk.

Reply:
yeah, multi-line comments would be required to be handled separlety.
Please create a ticket for these cases.

Reply:
https://yugabyte.atlassian.net/browse/DB-12598